clover_web.rb

```diff
@@ -184,6 +184,14 @@ def assets
       remove_all_active_sessions_except_current
     end
 
+    send_password_changed_email do
+      user = Account[account_id]
+      Util.send_email(email_to, "Ubicloud Account Password Changed",
```

Comment:
Is there any difference between using `email_to` vs `user.email`

Reply:
No difference for this example. https://github.com/jeremyevans/rodauth/blob/b44b4412fae55b9d97abaa34ab1e08434b161b90/lib/rodauth/features/email_base.rb#L30

For certain situations, like email change verification notifications, we must be cautious. We should also verify the value in other scenarios.